### PR TITLE
Add contributors

### PR DESCRIPTION
**Pull request by kaitejohnson**
@akeyel @ChiragKumar9 @hannahcohen4 @athowes Let me know if everything looks correct re names, email, ORCID ids!

### CODE REVIEW

**Comment by akeyel** [comment]
Looks good to me!

**Review by dylanhmorris — COMMENTED** [review]
Will wait until the contributors have all given a thumbs up before approving.

File: DESCRIPTION
Comment (zsusswein) [inline DESCRIPTION:1]: 😢

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**